kedro/framework/session/session.py

```diff
@@ -188,36 +186,11 @@ def create(  # pylint: disable=too-many-arguments
                 "Unable to get username. Full exception: %s", exc
             )
 
-        session._store.update(session_data)
-
-        # We need ConfigLoader and env to setup logging correctly
-        session._setup_logging()
         session_data.update(**_describe_git(session._project_path))
         session._store.update(session_data)
 
         return session
 
-    def _get_logging_config(self) -> dict[str, Any]:
```

Comment:
It makes me so happy to see these functions deleted 🤩

Reply:
Less is more!